week-1-practise/level-3/1-piping.js

```diff
@@ -11,6 +11,10 @@
 */
 
 var startingValue = 5;
+startingValue = multiply(startingValue, 2);
+startingValue = add(startingValue, 10);
+startingValue = format(startingValue);
+var result = startingValue;
```

Comment:
I think my advice about not reassigning function parameters applies here too. If we wanted to `console.log` the starting value (before it changes), we can't now. How could we change this to keep the `startingValue` the same?

Reply:
> As a learning exercise (you wouldn't do this normally), do all of the above in 1 line of code 

I think you missed this exercise? Can you combine all of the function calls into a single line? (I would put that as a separate bit of code)